front/store/index.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * The external imports
+ */
+import storage from "redux-persist/lib/storage";
+import { combineReducers } from "redux";
+import { configureStore } from "@reduxjs/toolkit";
+import {
+  persistReducer,
+  persistStore,
+  FLUSH,
+  REHYDRATE,
+  PAUSE,
+  PERSIST,
+  PURGE,
+  REGISTER,
+} from "redux-persist";
+
+/**
+ * The internal imports
+ */
+import user from "./user";
+
+const reducers = combineReducers({
+  user
+});
+
+const persistConfig = {
+  key: "root",
+  storage,
+  whitelist: ["theme"],
```

Comment:
is this ok ?
Cause I don't see  theme in combineReducers

Reply:
👍